openstreetmap/openstreetmap.php

```diff
@@ -12,13 +12,20 @@
 use Friendica\Core\Cache;
 use Friendica\Core\Config;
 use Friendica\Core\L10n;
+use Friendica\Core\System;
 use Friendica\Util\Network;
 
+const OSM_TMS = 'http://www.openstreetmap.org';
+const OSM_NOM =	'http://nominatim.openstreetmap.org/search.php';
+const OSM_ZOOM = 16;
+const OSM_MARKER = 0;
+
 function openstreetmap_install()
 {
 	Addon::registerHook('render_location', 'addon/openstreetmap/openstreetmap.php', 'openstreetmap_location');
 	Addon::registerHook('generate_map', 'addon/openstreetmap/openstreetmap.php', 'openstreetmap_generate_map');
 	Addon::registerHook('generate_named_map', 'addon/openstreetmap/openstreetmap.php', 'openstreetmap_generate_named_map');
+	Addon::registerHook('Map::getCoordinates', 'addon/openstreetmap/openstreetmap.php', 'openstreetmap_get_coordinates');
```

Comment:
has the notation of the hooks changed? If so, why not convert all called hooks to the new notation?

Reply:
The old ones had been named after the functions they came from. We renamed the functions, but the hook names stayed the same. This hook is new, so I named it after the function again.

Reply:
As it is new, could you then please add it to the addon/hook documentation.